fearless/Modules/Staking/StakingRewardPayouts/ViewModel/StakingPayoutViewModelFactory.swift

```diff
@@ -110,13 +110,14 @@ final class StakingPayoutViewModelFactory: StakingPayoutViewModelFactoryProtocol
         activeEra: EraIndex,
         payoutEra: EraIndex,
         historyDepth: UInt32,
-        eraCompletionTime: TimeInterval?,
+        eraCountdown: EraCountdown?,
         locale: Locale
     ) -> NSAttributedString {
         let eraDistance = historyDepth - (activeEra - payoutEra)
         let daysLeft = Int(eraDistance) / chain.erasPerDay
```

Comment:
Probably now we don't need `erasPerDay` because we can pass target era to `eraCompletionTime`